build/dataHash.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const fs = require('fs/promises');
+const path = require('path');
+const scraper = require('./scraper');
+const exportCache = require('../data/cache/.export.json');
+const locales = require('../config/locales.json');
+
+const exportKeyWhitelist = ['Manifest', 'DropChances', 'Patchlogs'];
+
+class DataHash {
+  constructor() {
+    this.exportCache = Object.fromEntries(
+      exportKeyWhitelist.filter((key) => key in exportCache).map((key) => [key, exportCache[key]])
+    );
+  }
+
+  isUptodate() {
```

Comment:
make sure to comment these, just to mention which caches they do and don't change/update/etc